app/uploaders/base_uploader.rb

```diff
@@ -2,6 +2,10 @@ class BaseUploader < CarrierWave::Uploader::Base
   include CarrierWave::BombShelter # limits size to 4096x4096
   include CarrierWave::MiniMagick # adds processing operations
 
+  FRAME_MAX = 1000
```

Comment:
I had a 768 frame gif that I used to test all of this and that GIF is really long. I don't see anyone needing more than 1000 frames, and worse case, if they do, they can upload it elsewhere and link to it.